### ISSUE TITLE
An ability to adjust the size of avatar

### ISSUE THREAD

**Issue by Loreos7** [ppy/osu-web#8353]
It will be useful to be able to select a specific area of the uploaded image for the profile avatar (adjusting scale esp.)

**Comment by KeziahMoselle** [comment]
I am planning to tackle this issue, anything I must know first?

**Comment by peppy** [comment]
I don't think so. Hopefully it can be slotted in without too much effect on the surroundings.

**Comment by KeziahMoselle** [comment]
Can we use a library for this? I am currently looking at these 2 libraries:
- https://github.com/react-cropper/react-cropper
- https://github.com/mosch/react-avatar-editor

Both are ~10kB minified/gzipped

**Comment by peppy** [comment]
I think it's okay. See how you go. Main concerns would be ensuring the chosen library is actively maintained (we have been forced to remain on an old react version due to an unmaintained dependency, which can be annoying).
